cmd/shfmt/main.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 var (
-	write  = flag.Bool("w", false, "write result to file instead of stdout")
-	list   = flag.Bool("l", false, "list files whose formatting differs from shfmt's")
-	indent = flag.Int("i", 0, "indent: 0 for tabs (default), >0 for number of spaces")
-	posix  = flag.Bool("p", false, "parse POSIX shell code instead of bash")
+	write       = flag.Bool("w", false, "write result to file instead of stdout")
+	list        = flag.Bool("l", false, "list files whose formatting differs from shfmt's")
+	indent      = flag.Int("i", 0, "indent: 0 for tabs (default), >0 for number of spaces")
+	posix       = flag.Bool("p", false, "parse POSIX shell code instead of bash")
+	showVersion = flag.Bool("V", false, "show version and exit")
```

Comment:
I realise `-v` is a bad idea since it can also be `verbose`, but I would use `-version` instead. Is `-V` popular in the wild? Haven't seen it yet.